app/controllers/application_controller.rb

```diff
@@ -56,15 +56,9 @@ def session_expired
     redirect_to session_expired_url
   end
 
-  def check_first_question_answered
```

Comment:
Could you add a note to the commit message about why this is being removed? Something about the filter question already being checked etc.

Reply:
:+1: